### DISCUSSION TITLE
HW submissions

### DISCUSSION THREAD

**Discussion by saewon1** [cooperrc/computational-mechanics#84] [Q&A]
So just to be clear, when submitting homework we are creating the HTML version from jupyter, then saving as PDF (for formatting reasons), then uploading the PDF to the google classroom submission? 

Do we have to upload anything to github?

**Comment by cooperrc** [comment]
> submitting homework we are creating the HTML version from jupyter, then saving as PDF (for formatting reasons), then uploading the PDF to the google classroom submission

Correct!

> Do we have to upload anything to github?

You can complete the course without ever pushing your work to your github.com fork (just by turning in pdfs). 

Your github fork is a permanent backup that you have access to even after the course is finished, so I encourage and help anyone who wants to use it, but I won't force you to use it (there's already a lot of new material for most people).
